tests/per_provider/normal_tests/create_destroy_key.rs

```diff
@@ -80,3 +97,56 @@ fn create_destroy_twice() -> Result<()> {
     client.destroy_key(key_name)?;
     client.destroy_key(key_name_2)
 }
+
```

Comment:
Do we have a test that a short RSA key, say, 512 bits, isn't allowed?

Reply:
I looked the API for `psa_generate_key`, I don't think it limits a specific size.

Reply:
Indeed. We might want to ask them for a suggestion for a minimum, for potentially marking `Deprecated` sizes in the future - doesn't make much sense to me for us to mark small elliptic curves as deprecated because of size but not doing the same for RSA - but maybe there's a reason for that!